src/VisualStudio/Core/Impl/SolutionExplorer/AnalyzerItem/AnalyzerItem.cs

```diff
@@ -9,83 +9,73 @@
 using Microsoft.VisualStudio.Imaging;
 using Microsoft.VisualStudio.Imaging.Interop;
 
-namespace Microsoft.VisualStudio.LanguageServices.Implementation.SolutionExplorer
+namespace Microsoft.VisualStudio.LanguageServices.Implementation.SolutionExplorer;
+
+internal partial class AnalyzerItem(
+    AnalyzersFolderItem analyzersFolder,
+    AnalyzerReference analyzerReference,
+    IContextMenuController contextMenuController) : BaseItem(GetNameText(analyzerReference))
 {
-    internal partial class AnalyzerItem : BaseItem
-    {
-        private readonly AnalyzersFolderItem _analyzersFolder;
-        private readonly AnalyzerReference _analyzerReference;
-        private readonly IContextMenuController _contextMenuController;
+    private readonly AnalyzersFolderItem _analyzersFolder = analyzersFolder;
+    private readonly IContextMenuController _contextMenuController = contextMenuController;
+
+    public AnalyzerReference AnalyzerReference { get; } = analyzerReference;
 
-        public AnalyzerItem(AnalyzersFolderItem analyzersFolder, AnalyzerReference analyzerReference, IContextMenuController contextMenuController)
-            : base(GetNameText(analyzerReference))
+    public override ImageMoniker IconMoniker
+    {
+        get
         {
-            _analyzersFolder = analyzersFolder;
-            _analyzerReference = analyzerReference;
-            _contextMenuController = contextMenuController;
+            return KnownMonikers.CodeInformation;
         }
+    }
 
-        public override ImageMoniker IconMoniker
+    public override ImageMoniker OverlayIconMoniker
+    {
+        get
         {
-            get
+            if (this.AnalyzerReference is UnresolvedAnalyzerReference)
             {
-                return KnownMonikers.CodeInformation;
+                return KnownMonikers.OverlayWarning;
             }
-        }
-
-        public override ImageMoniker OverlayIconMoniker
-        {
-            get
+            else
             {
-                if (_analyzerReference is UnresolvedAnalyzerReference)
-                {
-                    return KnownMonikers.OverlayWarning;
-                }
-                else
-                {
-                    return default;
-                }
+                return default;
             }
         }
+    }
 
-        public override object GetBrowseObject()
-        {
-            return new BrowseObject(this);
-        }
+    public override object GetBrowseObject()
+    {
+        return new BrowseObject(this);
+    }
 
-        public AnalyzerReference AnalyzerReference
```

Comment:
>AnalyzerReference [](http://example.com/codeflow?start=33&length=17)

could either of the properties directly below this be changed too?

Reply:
yes.  done.